### ISSUE TITLE
Function isArrayOfArrays translations AR, ES, FR, IT, HR

### ISSUE THREAD

**Issue by ilia-brykin** [ilia-brykin/aloha#123]
We need translations of Documentation for isArrayOfArrays: AR, ES, FR, IT, HR

URL: https://ilia-brykin.github.io/aloha/#/utils-is-array-of-arrays

**English words in a non-English translation should be highlighted for screen reader**
`<span lang="en"></span>` or `<strong lang="en"></strong>`

**Comment by ilia-brykin** [comment]
EN, DE, RU done.

**Comment by zdilarr** [comment]
HR done.

**Comment by shafedh** [comment]
AR, FR, IT, ES done.

**Comment by babeyeva** [comment]
OK
